packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/main/index.md

```diff
@@ -170,9 +170,55 @@ curl -X POST \
 
 > **Note**: You can copy the entire [response from the previous `GET /api/v1/apps/${id}/credentials/keys` request](#response-example) in to the `jwks.keys` array. The `created`, `lastUpdated`, and `expiresAt` properties are ignored in the POST request. Alternatively, you can remove them from the `jwks.keys` parameter body.
 
+#### Assign admin roles to the OAuth 2.0 service app
+
+> **Note:** Before the **Assign admin roles to public client apps** feature, the Super Administrator (`SUPER_ADMIN`) role was automatically assigned to service apps when they're created. You can now fine-tune the resources that a service app can access by assigning specific standard or custom admin roles. Ensure that the **Assign admin roles to public client apps** Self-Service feature is enabled in your org for granular role assignment. See [Manage Early Access and Beta features](https://help.okta.com/okta_help.htm?id=ext_Manage_Early_Access_features) for more information on the Okta Self-Service Feature Manager.
+
+Assign admin roles for every OAuth 2.0 service app that you create in the hub org. Service apps with assigned admin roles are constrained to the permissions and resources that are included in the role. This improves security for an org since it ensures that service apps only have access to the resources that are needed to perform their tasks. You can assign the [standard admin roles](https://help.okta.com/okta_help.htm?type=oie&id=ext-use-standard-roles) or a [custom admin role](https://help.okta.com/okta_help.htm?type=oie&id=ext-about-creating-custom-admin-roles) with permissions to specific resource sets.
+
+For the hub-and-spoke OAuth 2.0 Org2Org provisioning connection, Okta recommends that you assign the following [standard admin roles](https://help.okta.com/okta_help.htm?type=oie&id=ext-use-standard-roles):
```

Comment:
This link is not working for me. Should this link be used instead? https://help.okta.com/en-us/Content/Topics/Security/administrators-admin-comparison.htm

Reply:
The link should work now. Barbara just added the alias yesterday.
I'll request for another alias for the comparison page and use that link.